tools/travis/build.sh

```diff
@@ -21,7 +21,7 @@ $ANSIBLE_CMD apigateway.yml
 
 cd $ROOTDIR
 
-./gradlew distDocker -x :core:swift3Action:distDocker -x :core:pythonAction:distDocker -PdockerImagePrefix=testing
```

Comment:
I looked at the build times and building the containers did not add enough time to get close to the limit and it was in the noise relative to other builds. We will now run sniff tests against all the runtimes in Travis which is better than contributors not knowing they've broken something systemic for the previously ignored runtimes.

We can further cut the swift build time by putting up a base image in dockerhub. We should we do as part of your work on all images published to docker hub.